package.json

```diff
@@ -105,7 +106,8 @@
   },
   "jupyterlab": {
     "extension": true,
-    "outputDir": "jupyterlab_mystjs/labextension"
+    "outputDir": "jupyterlab_mystjs/labextension",
+    "disabledExtensions": ["@jupyterlab/notebook-extension:factory"]
```

Comment:
This ensures that _our_ content factory is made available to other extensions.

Reply:
this makes things much more compact but means there are some material changes, to the behaviour so far...

1. you can no longer access the standard markdown renderer at all. i.e. there is no markdown vs myst opeion on the Open With menu
2. there is no MyST icon, the notebooks use the standard notebook icon -- so there is no visual indication that MyST is in use -- other than myst markdown cells render properly.

on 1. previously you could opt out of MyST on a notebook-by-notebook basis, now you uninstall the extension to revert to Markdown only.

And 2. is not necessarily bad, and simplifies a lot of things, as we don't have to accommodate a concept different renderer in the launcher, the menus etc... you install the extension and MyST is everywhere.

So overall this is probably good -- simpler and puts myst on the track of replacing the standard markdown fully when it is installed.

Reply:
> you can no longer access the standard markdown renderer at all. i.e. there is no markdown vs myst opeion on the Open With menu

I spoke to @rowanc1 about some of these things, it would be good to restate them here!!

This disables the standard _notebook_ renderer. The consequence is that most other things now just work ™ with MyST. We can always disable this extension with `jupyter labextension disable ...`. It does mean you can't do so on a per-notebook basis, but as you state, I think this is probably a moot point.

Reply:
This is a lot nicer for workflows using jupytext -- you basically don't have to care about myst, you just work the way you always have!

From @agoose77 over slack:
> My take is that people are opting-in right now; having it installed & enabled means "I want to use it".

I think I am inclined to agree, and we can revert if we get feedback from our early adopters that this is too soon to do that. From a dev experience it is somewhat helpful to have an escape hatch if something goes wrong, but that is relatively straightforward with `jupyter labextension disable jupyterlab-mystjs`.